Sources/StreamChatUI/CommonViews/BaseViews.swift

```diff
@@ -49,11 +50,19 @@ public extension Customizable where Self: UIViewController {
     }
 }
 
+extension UIConfigProvider where Self: _View {
+    public func uiConfigDidRegister() {
+        if isInitialized {
+            log(.error, message: "UIConfig assigned after the view is initialized.")
```

Comment:
Let's throw an assertion failure here. This is a serious error. You can use `log.assertionFailure` for that. The message can be something like:
```
`UIConfig` was assigned after the view has been already initialized. This is most likely caused by assigning the custom `UIConfig` instance after the view has been added to the view hierarchy, or after the view's subviews have been initialized already. This is undefined behavior and should be avoided.
```

Reply:
✅